README

```diff
@@ -56,6 +56,12 @@ vdr -P 'satip -s <ipaddress>|<model>|<description>;...'
 vdr -P 'satip -s 192.168.0.1|DVBS2-2,DVBT2-2|Octo1'
 vdr -P 'satip -s 192.168.0.1|DVBS2-4|Octo1;192.168.0.2|DVBT2-4|Octo2'
 
+The plugin accepts a "--portrange" (-p) command-line parameter, it can
+be used to manually specify the RTP & RTCP port range. This makes it 
+possible to use the plugin through a NAT (e.g. a docker bridged network)
+A minimum of 2 ports per device are needed.
+Number of ports = base port + number of devices * 2 - 1
```

Comment:
The plugin accepts a "--portrange" (-p) command-line parameter, it can
be used to manually specify the RTP & RTCP port range. This makes it 
possible to use the plugin through a NAT (e.g. a docker bridged network...)
A minimum of 2 ports per device is required.

?